### PR TITLE
1.0.0-rc

### PR DESCRIPTION
**Pull request by wuputah**
@JerrySievert can you review the changelog content for accuracy?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
 
+## 1.0.0-rc
+
+In addition to bug fixes, this release includes significant vectorization optimization for aggregates (eg.g COUNT and SUM), enabled new index types, and added `pg_hint_plan`.
+
+* vectorization of direct aggregates ([#143][])
+* fix an inefficiency with vacuum if there is only one stripe ([#143][])
+* enable `O3` compilation optimizations ([#143][])
+* disable parallelism for `CREATE TABLE ... AS` queries ([#138][])
+* allow gin, gist, spgist, and rum indexes ([#133][])
```
Comment (JerrySievert) [inline CHANGELOG.md:11]: with columnar

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+* add [pg_hint_plan][] extension ([#134][])
+* fix `columnar.alter_table_set_access_method` bug with disallowing foreign keys ([#136][])
```
Comment (JerrySievert) [inline CHANGELOG.md:13]: maybe a way to make this sound a little more clear as to what was done?

Comment (wuputah) [reply CHANGELOG.md]: changed to:
>  fix `columnar.alter_table_set_access_method` to correctly disallow conversion of heap tables that have foreign keys

**Review by JerrySievert — APPROVED** [review]
lgtm, don't think we're missing anything, at least nothing is jumping out at me.

**Pull request merged**